ReadMe.md

```diff
@@ -58,7 +58,10 @@ To edit | 編集（する） | へんしゅう |
 Automatic | 自動 | じどう |
 Screen | 画面 | がめん |
 Video | 動画 | どうが |
+Picture | 写真 |　しゃしん
+Download | ダウンロード
```

Comment:
perhaps we should add in the notes if the work can be used as a noun as well
we have a few examples of this already, would be painful to split them all up